Gemfile

```diff
@@ -11,4 +11,4 @@ group :development do
   gem 'omniauth-twitter'
 end
 
-gem 'devise', '>= 4.8.0'
+gem 'devise', '>= 4.9.0.alpha', github: 'heartcombo/devise'
```

Comment:
As you may know, Devise 4.9.0 has been released. 
https://github.com/heartcombo/devise/releases/tag/v4.9.0